drivers/mrf24j40/mrf24j40.c

```diff
@@ -81,6 +81,7 @@ bool mrf24j40_cca(mrf24j40_t *dev)
     uint8_t tmp_rssi;
 
     mrf24j40_assert_awake(dev);
+    mrf24j40_enable_lna(dev);
```

Comment:
I'm unsure what I like better: `#ifdef` around the function call or inside (as you did). I tend to prefer outside because it wouldn't call the function if `MRF24J40_USE_EXT_PA_LNA` is zero  and it makes it directly visible, although the code might get uglier. What was you motivation to implement it inside?

Reply:
> What was you motivation to implement it inside?

Exactly to keep the code cleaner - and only need one `#ifdef` instead of two.